src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../index.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends React.Component {
```

Comment:
- [ ] I kindly suggest that you use make your `Calculator` component a functional component instead of making use of class.